remix.init/index.js

```diff
@@ -0,0 +1,136 @@
+const inquirer = require("inquirer");
+const fs = require("fs/promises");
+const { join } = require("path");
+const PackageJson = require("@npmcli/package-json");
+
+const foldersToExclude = [".github"];
+
+const edgeFilesToCopy = [
+  ["README-edge.md", "README.md"],
+  ["netlify-edge.toml", "netlify.toml"],
+  ["server.js"],
+  ["remix.config.js"],
+  ["vscode.json", join(".vscode", "settings.json")],
+];
+
+const filesToCopy = [["README.md"], ["netlify.toml"]];
+
+const filesToModify = ["app/entry.server.tsx", "app/root.tsx"];
+
+async function modifyFilesForEdge(files, rootDirectory) {
+  const filePaths = files.map((file) => join(rootDirectory, file));
+  const contents = await Promise.all(
+    filePaths.map((path) => fs.readFile(path, "utf8"))
+  );
+
+  await Promise.all(
+    contents.map((content, index) => {
+      const newContent = content.replace(
+        /@remix-run\/node/g,
+        "@netlify/remix-runtime"
+      );
+      return fs.writeFile(filePaths[index], newContent);
+    })
+  );
+}
+
+async function copyTemplateFiles(files, rootDirectory) {
+  for (const [file, target] of files) {
+    await fs.copyFile(
+      join(rootDirectory, "remix.init", file),
+      join(rootDirectory, target || file)
+    );
+  }
+}
+
+async function updatePackageJsonForEdge(directory) {
+  const packageJson = await PackageJson.load(directory);
+  const {
+    dependencies: {
+      "@remix-run/netlify": _netlify,
+      "@remix-run/node": _node,
+      ...dependencies
+    },
+    scripts,
+    ...restOfPackageJson
+  } = packageJson.content;
+
+  packageJson.update({
+    // dev script is the same as the start script for Netlify Edge, "cross-env NODE_ENV=production netlify dev"
+    scripts: {
+      ...scripts,
+      dev: scripts["start"],
+    },
+    ...restOfPackageJson,
+    dependencies: {
+      ...dependencies,
+      "@netlify/edge-functions": "^2.0.0",
+      "@netlify/remix-edge-adapter": "1.0.0",
```

Comment:
I wonder if there's a way to ensure this is kept up to date.

Reply:
We could put `*` like the others

Reply:
Those work because the publish script automatically replaces that with the latest version. That wouldn't work for a js file

Reply:
We could query the npm registry with a GH action cron job and update the version.

`const { version } = await fetch("https://registry.npmjs.org/@netlify/remix-edge-adapter/latest");`

It's out of the scope of this PR, but I can create an issue for it.

Reply:
One way might be to put them in dev dependencies in the init folder, load the version from there, and let renovate keep it up to date